scout/parse/variant/variant.py

```diff
@@ -193,6 +193,16 @@ def parse_variant(variant, case, variant_type='clinical',
     if str_status:
         parsed_variant['str_status'] = str(str_status)
 
+    # str normal max - max number of repeats to call an STR variant normal
+    str_normal_max = variant.INFO.get('STR_NORMAL_MAX')
```

Comment:
Nice! Can we get a test as well?

Reply:
Coveralls says ⛔️. 😉

Reply:
I've added tests for STR parsing. I had to add new STR vcfs that were annotated with the latest Stranger version. Hopefully the tests are somewhat meaningful. I basically copied them from the SNV/SV tests.